### ISSUE TITLE
Server error 1.16.5 when streaming emote

### ISSUE THREAD

**Issue by Raipus** [KosmX/emotes#177]
I have latest version of Emotecraft on client ( emotecraft-for-MC1.16.5-2.1-SNAPSHOT-build.8-fabric.jar ). All players on my server use this version because of  a "SECURITY VULNERABILITY IN JAVA EDITION" ( We updated to Fabric 0.12.9 ). We ask for help.

**Console Error:**

```
[14:58:59 INFO]: [EMOTECRAFT] streaming emote
[14:58:59 WARN]: Plugin emotecraft v2.0.6 generated an exception whilst handling plugin message
java.lang.NoSuchMethodError: java.nio.ByteBuffer.position(I)Ljava/nio/ByteBuffer;
	at io.github.kosmx.emotes.common.network.EmotePacket.read(EmotePacket.java:135) ~[?:?]
	at io.github.kosmx.emotes.bukkit.BukkitWrapper.receivePluginMessage(BukkitWrapper.java:80) ~[?:?]
	at io.github.kosmx.emotes.bukkit.BukkitWrapper$$Lambda$3674/1602246005.onPluginMessageReceived(Unknown Source) ~[?:?]
	at org.bukkit.plugin.messaging.StandardMessenger.dispatchIncomingMessage(StandardMessenger.java:455) ~[patched_1.16.5.jar:git-Paper-792]
	at net.minecraft.server.v1_16_R3.PlayerConnection.a(PlayerConnection.java:3084) ~[patched_1.16.5.jar:git-Paper-792]
	at net.minecraft.server.v1_16_R3.PacketPlayInCustomPayload.a(SourceFile:44) ~[patched_1.16.5.jar:git-Paper-792]
	at net.minecraft.server.v1_16_R3.PacketPlayInCustomPayload.a(SourceFile:9) ~[patched_1.16.5.jar:git-Paper-792]
	at net.minecraft.server.v1_16_R3.PlayerConnectionUtils.lambda$ensureMainThread$1(PlayerConnectionUtils.java:35) ~[patched_1.16.5.jar:git-Paper-792]
	at net.minecraft.server.v1_16_R3.PlayerConnectionUtils$$Lambda$4275/389405014.run(Unknown Source) ~[?:?]
	at net.minecraft.server.v1_16_R3.TickTask.run(SourceFile:18) ~[patched_1.16.5.jar:git-Paper-792]
	at net.minecraft.server.v1_16_R3.IAsyncTaskHandler.executeTask(IAsyncTaskHandler.java:136) ~[patched_1.16.5.jar:git-Paper-792]
	at net.minecraft.server.v1_16_R3.IAsyncTaskHandlerReentrant.executeTask(SourceFile:23) ~[patched_1.16.5.jar:git-Paper-792]
	at net.minecraft.server.v1_16_R3.IAsyncTaskHandler.executeNext(IAsyncTaskHandler.java:109) ~[patched_1.16.5.jar:git-Paper-792]
	at net.minecraft.server.v1_16_R3.MinecraftServer.bb(MinecraftServer.java:1271) ~[patched_1.16.5.jar:git-Paper-792]
	at net.minecraft.server.v1_16_R3.MinecraftServer.executeNext(MinecraftServer.java:1264) ~[patched_1.16.5.jar:git-Paper-792]
	at net.minecraft.server.v1_16_R3.IAsyncTaskHandler.awaitTasks(IAsyncTaskHandler.java:119) ~[patched_1.16.5.jar:git-Paper-792]
	at net.minecraft.server.v1_16_R3.MinecraftServer.sleepForTick(MinecraftServer.java:1225) ~[patched_1.16.5.jar:git-Paper-792]
	at net.minecraft.server.v1_16_R3.MinecraftServer.w(MinecraftServer.java:1139) ~[patched_1.16.5.jar:git-Paper-792]
	at net.minecraft.server.v1_16_R3.MinecraftServer.lambda$a$0(MinecraftServer.java:291) ~[patched_1.16.5.jar:git-Paper-792]
	at net.minecraft.server.v1_16_R3.MinecraftServer$$Lambda$3058/626236019.run(Unknown Source) ~[?:?]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_51]
```

**Comment by KosmX** [comment]
This appears to be a Bukkit log, not a Fabric.  

Emotecraft 2.1 snapshots are **not** fully compatible with 2.0 versions. Including the Bukkit plugin.  

Try it with a snapshot plugin build:   
https://github.com/KosmX/emotes/releases/tag/2.1-SNAPSHOT-build.8

**Comment by KosmX** [comment]
Also!
You **have** to use Fabric 0.12.11 to fix the vulnerability on 1.16.5 or on older versions.
(but clients should be fine)

**Comment by Raipus** [comment]
Thanks, that helped!
